exercises/error-handling/example.rs

```diff
@@ -0,0 +1,81 @@
+use std::cmp;
+
+#[derive(Eq,PartialEq,Debug)]
+pub enum IceCream {
+    Chocolate,
+    Vanilla,
+    Strawberry,
+}
+
+impl IceCream {
+    /// Convert various string versions of ice cream flavors to a
+    /// proper IceCream enum or return Error
+    pub fn get_flavor(request: &str) -> Result<IceCream, &'static str> {
```

Comment:
any reason why this function is not present in the stub, even though the stub defines all other functions that the test would expect?

Reply:
Never mind, this is explained.

> I decided to remove the impl for the Enum, as this may be new functionality for a user who has only seen impl for structs. I mention it in the hints.md, so they don't get stuck, but want them to research a little if this is new to them.

I will think about whether I agree with it.